src/lib/components/Banner.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+	export let message = 'Welcome to our website!'
+	export let bgColor = '#4CAF50' // Default background color
+
+	let visible = true // Initially, the banner is visible
+</script>
+
+{#if visible}
+	<div class="banner" style="--bg-color: {bgColor};">
```

Comment:
Try to avoid inline styles if possible. In this case, I don't think we'll ever modify the banner color so we don't need the bgColor prop anyway.

Reply:
done